index.js

```diff
@@ -49,7 +69,16 @@ module.exports = (babel, options) => {
     presets: [
       [
         require('@babel/preset-env').default,
-        {include, loose, modules, targets, bugfixes: true, corejs: 3, useBuiltIns: 'entry'},
+        {
+          include,
+          loose,
+          modules,
+          targets,
+          bugfixes: true,
+          corejs: 3,
+          useBuiltIns: 'entry',
+          browserslistEnv: env.includes('modern') ? 'production-modern' : undefined, // defaults to [production]
```

Comment:
Generally, this preset allows overriding most of its behavior, and I think we should allow the same here. Can you add a `browserslistEnv` option, and use that if specified, otherwise defaulting to the current logic? See the other forwarded options like `loose` and `modules` for examples of this.

Reply:
Resolved in https://github.com/kensho-technologies/babel-preset/pull/88/commits/743a8a0d8c164d24a97e70913deb35b3ab3b429f while I was touching that anyway.